spec/spec_helper.rb

```diff
@@ -51,7 +51,7 @@
 
 Capybara.register_driver :chrome do |app|
   options = Selenium::WebDriver::Chrome::Options.new(
-    args: %w[headless disable-gpu no-sandbox window-size=1280,768]
```

Comment:
:eyes:

Reply:
These last 6 commits feel a bit rushed... is it necessary to put them in this PR, or can they go in a new one?

Reply:
ah, no, not rushed (I spent about an hour on this today), I just forgot to revert this locally before committing :-(
I deleted it now.

Reply:
This is boy scout rule for me and also rubocop, checkout spec was long and is still a bit messy, I had to make it better before adding specs necessary for this PR. I would prefer this is part of this PR (even if it takes longer to review). The commits are just moving code around: I created a separate file for all guest checkout tests, that's it. I have then simplified code by removing teh variables and factories that were not being used in each of the parts.

Reply:
anyway, the answer to your question is no, I can create a separate PR. I have to go now, I'll do that tomorrow.

Reply:
> I had to make it better before adding specs necessary for this PR. I would prefer this is part of this PR

That sounds like the answer is "yes" then?

Reply:
I am sorry, I meant your first question :rofl: 

>>> is it 100% necessary to include them in this PR

no

>>> can they go in a new one?

yes

:-)